testing/integration/kubernetes_agent_standalone_test.go

```diff
@@ -60,7 +62,11 @@ func TestKubernetesAgentStandaloneKustomize(t *testing.T) {
 		Local: false,
 		Sudo:  false,
 		OS: []define.OS{
-			{Type: define.Kubernetes},
+			{Type: define.Kubernetes, DockerImage: "docker.elastic.co/beats/elastic-agent"},
+			{Type: define.Kubernetes, DockerImage: "docker.elastic.co/beats/elastic-agent-complete"},
+			{Type: define.Kubernetes, DockerImage: "docker.elastic.co/beats/elastic-agent-ubi"},
```

Comment:
For 9.0 the elastic-agent and elastic-agent-ubi will eventually be the same, so this is redundant there but not in 8.x where Ubuntu will continue to exist.

Reply:
I can do a followup in main to merge those into the same.